buy_together_app/tests/test_sign_up.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+from django.template.loader import get_template
+from supplier.models import Supplier
+from client.models import Client
+from django.contrib.auth.models import User
+
+
+@pytest.mark.django_db()
+class TestSignupPage:
+    def test_signup_template(self):
+        get_template('buy_together_app/sign_up.html')
+
+    def test_signup_supplier(self, client):
+        response = client.post(
+            '/signup/supplier', {'username': "testSignUp",
+                                 'password': "pokemon123",
+                                 'first_name': "ash",
+                                 'last_name': "katchamp",
+                                 'business_name': "pokemon inc"}
+        )
+        assert User.objects.get(username="testSignUp") in list(User.objects.all())
```

Comment:
No need to add `list(...)` here. 
```suggestion
        assert User.objects.get(username="testSignUp") in User.objects.all()
```

Reply:
Fixed